app/models/github_issue.rb

```diff
@@ -41,7 +41,8 @@ def retrieve_and_save_issue(url)
 
       # despite the counter intuitive name `.markdown` returns HTML rendered
       # from the original markdown
-      issue.processed_html = Github::OauthClient.new.markdown(issue.issue_serialized[:body])
+      issue.processed_html =
```

Comment:
this is shorter code but might be a little wasteful (we post something to GH api when we know what to expect back).

A perfectly reasonable alternative would be

```ruby
issue.processed_html = if issue.issue_serialized[:body].present?
  Github::OauthClient.new.markdown(issue.issue_serialized[:body])
else
  ""
end
```

Reply:
I think I prefer the slightly longer alternative version.